tests/cpp/test_multi_matmul_scheduler.cpp

```diff
@@ -201,6 +201,10 @@ class MultiMatmulSchedulerMatchTest
               << " to original IterDomain " << id_orig->toString();
     std::string suffix = suffix_ss.str();
     EXPECT_EQ(id_orig->getIterType(), id_new->getIterType()) << suffix;
+    // ParallelType checking is disabled altogether for now. We now check that
+    // the compiled code matches which verifies that the loop groups have the
+    // same ParallelType.
```

Comment:
To be clear, all tests pass currently using the cuda kernel comparison, but the scheduled IR differs in the `ParallelType` of some tensors compared to the original matmul scheduler.